custom_dc/custom_data.md

```diff
@@ -230,8 +230,8 @@ The `sources` section is optional. It encodes the sources and provenances associ
 
 ## Load local custom data
 
-To load custom data uploaded to Google Cloud, see instead [Pointing the local Data Commons site to the Cloud data](testing_cloud.md) for procedures.
-
+To load custom data uploaded to Google Cloud, see instead [Pointing the local Data Commons site to the Cloud data](/custom_dc/testing_cloud.html) for procedures.
```

Comment:
It looks like this line got doubled here. Please fix before merging.

Reply:
I'm gonna just merge this and will fix later. I've had so much aggravation with github today I don't want to lose more changes!